jujugui/static/gui/src/app/components/expanding-row/expanding-row.js

```diff
@@ -116,7 +116,7 @@ YUI.add('expanding-row', function() {
     _resize: function() {
       const expanded = this.state.expanded;
       this.setState({styles: {
-        height: expanded ?  this.refs.inner.offsetHeight + 'px' : '0px',
+        height: expanded ?  'auto' : '0px',
```

Comment:
I believe this change will break the css animations. Lets chat about why this change was necessary.